astropy/nddata/utils.py

```diff
@@ -519,9 +519,10 @@ def __init__(self, data, position, size, wcs=None, mode='trim',
             units of pixels.  ``size`` can also be a
             `~astropy.units.Quantity` object or contain
             `~astropy.units.Quantity` objects.  Such
-            `~astropy.units.Quantity` objects must be in pixel or angular
-            units.  See the ``mode`` keyword for additional details on
-            the final cutout size.
+            `~astropy.units.Quantity` objects must be in pixel or
+            angular units.  If ``size`` contains any floats, they will
```

Comment:
Ah, sorry I misunderstood!  Yes, you're correct.  I'll add this to the docs:  "`size` will be converted to an integer number of pixels, rounding to the nearest pixel."